addon/helpers/max.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
```

Comment:
Can you please destructure here and do `const {Helper} = Ember;` rather than using `Ember.Helper`?